File: src/com/bean/Labor.java
package com.bean;

public class Labor {
private int laborcount;
private int sal;
public Labor(int laborcount, int sal) {
	super();
	this.laborcount = laborcount;
	this.sal = sal;
}
public int getLaborcount() {
	return laborcount;
}
public void setLaborcount(int laborcount) {
	this.laborcount = laborcount;
}
public int getSal() {
	return sal;
}
public void setSal(int sal) {
	this.sal = sal;
}
@Override
public String toString() {
	return "Labor [laborcount=" + laborcount + ", sal=" + sal + "]";
}

}

File: src/com/bean/Labor.go
package bean

import (
	"fmt"
)

type Labor struct {
	laborcount int
	sal        int
}

func CreateLabor(laborcount int, sal int) Labor {
	return Labor{laborcount: laborcount, sal: sal}
}
func (l Labor) getLaborcount() int {
	return l.laborcount
}
func (l Labor) setLaborcount(laborcount int) {
	l.laborcount = laborcount
}
func (l Labor) getSal() int {
	return l.sal
}
func (l Labor) setSal(sal int) {
	l.sal = sal
}
func (l Labor) String() string {
	return fmt.Sprintf("Labor [Labor=%d, sal=%d]", l.laborcount, l.sal)
}
